### ISSUE TITLE
Serious bug(s) in Show Instrument view

### ISSUE THREAD

**Issue by smk78** [mantidproject/mantid#35501]
Am surprised this has not already been picked up, but...

**Describe the bug**
Trying to use alternate projections and/or ROI tools in Show Instrument view causes the instrument view to go blank meaning the functionality is actually unusable!

**To Reproduce**
Load a nxs file (example here SANS2D #78824)
Right-click > Show Instrument
Adjust the colour scale to show something!
![image](https://user-images.githubusercontent.com/10938679/235913161-e305a0d2-558d-4d39-880b-22fd9ef4abff.png)
Click the top button in the Render tab  (currently showing Full 3D) and select Cylindrical Y (this projection is required to properly create a detector mask)
![image](https://user-images.githubusercontent.com/10938679/235913659-e3a54a63-6994-4bec-85de-64ffbdf4087d.png)
Switch to the Draw tab: the image vanishes!
![image](https://user-images.githubusercontent.com/10938679/235913876-721ad16a-8340-4fa0-b091-03c388f6f2d3.png)
Return to the Render tab > Reset View
Switch back to the Draw tab: this time the image is there!
![image](https://user-images.githubusercontent.com/10938679/235914379-0e5a1b88-ca7f-49bf-b8cf-2e72dc677f13.png)
Select a Navigation tool from the icons: eg, the magnifying glass to expand the oval ring
Draw a box around the ring: the image vanishes again!
![image](https://user-images.githubusercontent.com/10938679/235914770-14b06d03-4cac-4e3e-abbd-b5c5c73ccced.png)

Basically, the ROI functionality in Show Instrument needed for things like detector masking is currently broken.

**Expected behavior**
The instrument projections should not go blank and it should be possible to use the tools to select/expand/manipulate ROIs.

**Platform/Version (please complete the following information):**
 - OS: W10 Enterprise
 - OS Version: Build 19044
 - Mantid Version: 6.6.0, 6.5.0,...5.0.0... (versions I just happen to have installed)

**Comment by gemmaguest** [comment]
I've seen odd behaviour like this from time to time when running on my remote machine. I figured it was a timing issue to do with the widget redraw happening at slightly the wrong time. Usually any operation that causes a redraw (like zoom/pan/change tab) will make the display appear again. I think even moving another window over the widget can cause it to redraw. @smk78 does it sound like it could be something similar?
